### ISSUE TITLE
[Package Request] - provide system-logos-httpd

### ISSUE THREAD

**Issue by bparsons** [amazonlinux/amazon-linux-2022#136]
nginx is not able to install due to a missing dependency system-logos-httpd

As a workaround, installing an [RPM from fedora](https://rpmfind.net/linux/fedora/linux/updates/34/Everything/x86_64/Packages/f/fedora-logos-httpd-34.0.4-1.fc34.noarch.rpm) provides the dependency, but you will probably want to create a logo package for amzn2022

**Comment by stewartsmith** [comment]
We even have a logo now!

We had a slight out-of-order problem with cutting the generic-logos package and introducing the Amazon Linux logos package which has resulted in this (kind of embarrassing)  bug.

We're working on getting a fix for the next release though.

**Comment by kotofos** [comment]
Just exact error message for a search engines
```
dnf install nginx 

Error: 
Problem: conflicting requests 
 - nothing provides system-logos-httpd needed by nginx-1:1.20.1-8.amzn2022.0.2.x86_64
 ```

**Comment by stewartsmith** [comment]
Fixed in 2022.0.20220728
